README.md

```diff
@@ -1 +1,104 @@
-# image-scan-example
\ No newline at end of file
+# Image-Scan-Example
+
+#### PREREQUISITES: 
+
+All of the CI examples rely on an existing DockerHub repo that you control and have the ability to push the container images being built. This repo should exist before getting started. This repo value will be used in the `CONTAINER_REPO` variable throughout the examples. You will also need a personal access token to be used for the DockerHub password in all pipelines.
+
+## CI Pipeline Examples
+
+The pipeline examplea use the Dockerfile inside this directory for build purposes.
```

Comment:
> this directory

Which directory? Or do you mean the repository?

Reply:
changed to repository